code/go/internal/validator/semantic/validate_minimum_kibana_version.go

```diff
@@ -52,11 +58,25 @@ func validateMinimumKibanaVersion(packageType string, packageVersion semver.Vers
 		return nil
 	}
 
-	if kibanaVersionConditionIsGreaterThanOrEqualTo8_8_0(kibanaVersionCondition) {
+	if kibanaVersionConditionIsGreaterThanOrEqualTo(kibanaVersionCondition, "8.8.0") {
 		return nil
 	}
 
-	return errors.New("Warning: conditions.kibana.version must be ^8.8.0 or greater for non experimental input packages (version > 1.0.0)")
+	return errors.New("conditions.kibana.version must be ^8.8.0 or greater for non experimental input packages (version > 1.0.0)")
+}
+
+func validateMinimumKibanaVersionRuntimeFields(fsys fspath.FS, packageVersion semver.Version, kibanaVersionCondition string) error {
+
+	errs := validateFields(fsys, validateNoRuntimeFields)
+	if len(errs) == 0 {
+		return nil
+	}
+
+	if kibanaVersionConditionIsGreaterThanOrEqualTo(kibanaVersionCondition, "8.9.0") {
```

Comment:
As there is no failure if a package that contains a runtime field is installed in kibana versions that do not support those kind of fields, this validation rule ensures that the packages must have at least a given kibana version.

What version should I set here @kpollich ? I was thinking that maybe it is 8.9 ?

Reply:
Talked offline with @kpollich , Elastic stack / Kibana `8.9.0` version is going to be set as minimum version for runtime fields.